examples/custom-metrics-tuning/values.yaml

```diff
@@ -10,29 +11,38 @@ externalServices:
 
 metrics:
   alloy:
-    metricsTuning: # Will filter to the metrics that will populate the Grafana Alloy integration
+    metricsTuning:
+      # Will filter to the metrics that will populate the Grafana Alloy integration
       useIntegrationAllowList: true
```

Comment:
Something broke here, since the list of allowed metrics in the `metrics.alloy` above has lost the integration list, and it's just the default k8s monitoring list.

Reply:
This has been resolved.  Due to adding the `---` at the start of the files: 

- `./charts/k8s-monitoring/default_allow_lists/alloy_integration.yaml`
- `./charts/k8s-monitoring/default_allow_lists/alloy.yaml`
- `./charts/k8s-monitoring/default_allow_lists/node_exporter_integration.yaml`
- `./charts/k8s-monitoring/default_allow_lists/node_exporter.yaml`

The templating was seeing 2 yaml files instead of a single.  I updated the template `allowList` template function in both of these files: 

- `./charts/k8s-monitoring/templates/alloy_config/_alloy.alloy.txt`
- `./charts/k8s-monitoring/templates/alloy_config/_node_exporter.alloy.txt`